src/Microsoft.ML.Data/Microsoft.ML.Data.csproj

```diff
@@ -5,6 +5,7 @@
     <IncludeInPackage>Microsoft.ML</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>CORECLR</DefineConstants>
+    <BaseIntermediateOutputPath>..\..\bin\obj\AnyCPU.Debug\Microsoft.ML.Data</BaseIntermediateOutputPath>
```

Comment:
This should be reverted.

Reply:
Indeed. No idea where that came up, thanks @eerhardt .